pkg/collector/corechecks/system/iostats_nix.go

```diff
@@ -75,6 +76,9 @@ func (c *IOCheck) nixIO() error {
 		tagbuff.WriteString("device:")
 		tagbuff.WriteString(device)
 		tags := []string{tagbuff.String()}
+		if len(ioStats.Label) > 0 {
+			tags = append(tags, fmt.Sprintf("device-label:%s", ioStats.Label))
```

Comment:
I think tags can't include dashes (`-`)

Reply:
From the doc it can contain `minuses`: https://docs.datadoghq.com/tagging/#defining-tags

Reply:
unless there's a specific reason to use a `minus` here, I'd vote in favor of using an underscore since that's the convention in most of our integrations